roles/internal/alpaca/defaults/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+alpaca_from_source: yes
+
+alpaca_version: master
+alpaca_clone_directory: /opt/alpaca
+
+alpaca_karaf_repos: []
+
+alpaca_features:
+  - islandora-http-client
+  - islandora-connector-broadcast
+  - islandora-indexing-triplestore
+  - islandora-indexing-fcrepo
+
+alpaca_karaf_dir: /opt/karaf
+alpaca_karaf_etc_dir: "{{ alpaca_karaf_dir }}/etc"
+alpaca_local_mvn_path: /home/{{ alpaca_user }}/.m2/repository
+alpaca_user: karaf
+
+alpaca_settings:
+  - pid: ca.islandora.alpaca.connector.broadcast
+    settings:
+      input.stream: broker:queue:islandora-connector-broadcast
```

Comment:
Should be `activemq:queue:islandora-connector-broadcast`.  Anytime you see `broker:*` it's just a placeholder for `activemq` or whatever other queuing technology camel supports.